eloquentjs/11_3_real_promises/src/fileToArray.js

```diff
@@ -0,0 +1,3 @@
+const fileToArray = file => file.split('\n').filter(j => j.length > 0);
```

Comment:
I'd probably combine this and textFile into a module named something like logFileHelpers